src/Microsoft.Extensions.ApiDescription.Client/build/Microsoft.Extensions.ApiDescription.Client.targets

```diff
@@ -248,7 +264,9 @@
 
     <ItemGroup>
       <ServiceFileReference Remove="@(ServiceFileReference)" />
-      <ServiceFileReference Include="@(_Temporary)" />
+      <ServiceFileReference Include="@(_Temporary)">
+        <RelativeOutputPath>$([MSBuild]::MakeRelative($(MSBuildProjectDirectory), %(_Temporary.OutputPath)))</RelativeOutputPath>
```

Comment:
At the project level, `OutputPath` is relative, and things that need a full path construct it using the project directory. Might be something to consider.

Reply:
Changed my mind and made `%(OutputPath)` relative. Don't really need an absolute path because code generators run in the client project.